pom.xml

```diff
@@ -167,6 +167,17 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials-binding</artifactId>
     </dependency>
+    <!-- BouncyCastle API Plugin -->
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>bouncycastle-api</artifactId>
+    </dependency>
+    <!-- Apache Mina SSHD Core-->
+    <dependency>
+      <groupId>org.apache.sshd</groupId>
+      <artifactId>sshd-core</artifactId>
+      <version>2.7.0</version>
+    </dependency>
```

Comment:
I'm not sure if this is required (or even safe), but I excluded the transitive dependencies that arrived due to the inclusion of sshd-common.
```suggestion
      <version>2.7.0</version>
      <exclusions> <!-- prevent inclusion in hpi as a transient dependency -->
        <exclusion>
          <groupId>org.slf4j</groupId>
          <artifactId>jcl-over-slf4j</artifactId>
        </exclusion>
        <exclusion>
          <groupId>org.slf4j</groupId>
          <artifactId>slf4j-api</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

Reply:
should be fine